### PR TITLE
7 vote on article after login

### PR DESCRIPTION
**Pull request by DexterGundar**

### CODE REVIEW

File: src/api.js [hunk 1]
```diff
@@ -17,6 +17,17 @@ export const getArticleById = (article_id) =>{
     })
 }
 
+export const patchArticleVotes = (article_id, votesToPatch) => {
+    const voteUpdate = { inc_votes: votesToPatch };
+    return newsApi.patch(`/articles/${article_id}`, voteUpdate)
+    .then(function (response) {
+        console.log(response);
```
Comment (RichardNeat) [inline src/api.js:24]: pls remove console log

File: src/api.js [hunk 2]
```diff
@@ -17,6 +17,17 @@ export const getArticleById = (article_id) =>{
...
+    })
+    .catch(function (error) {
+            console.log(error);
```
Comment (RichardNeat) [inline src/api.js:27]: ditto the above :)

**Review by RichardNeat — COMMENTED** [review]
Nice, just a couple of suggestions:
- We still have the comments issue where it's not clear how a user can access comments. This will need signposting somehow if you still want this on a separate page.
- Consider some error messaging here and we will undo the optimistic rendering. Currently there is no messaging to say it has failed and everything looks like it has gone fine.

File: src/api.js (hunk 1 above)
Comment (DexterGundar) [reply src/api.js:24]: There is a sentence: Click to login for voting and comments: right where to click. I thought that would be enough...

**Pull request merged**